RecoTracker/LSTCore/src/alpaka/Event.dev.cc

```diff
@@ -1,6 +1,8 @@
 #include "Event.h"
 
-void SDL::Event<SDL::Acc>::init(bool verbose) {
+using namespace ALPAKA_ACCELERATOR_NAMESPACE;
+
+void ALPAKA_ACCELERATOR_NAMESPACE::SDL::Event<Acc3D>::init(bool verbose) {
```

Comment:
is this needed considering the `using namespace` above?

Reply:
or do we now have `ALPAKA_ACCELERATOR_NAMESPACE::SDL` and global `SDL` ? this is confusing; better not do it

Reply:
It was needed because it said it was ambiguous, but I'm getting rid of the nested namespaces, so it will be fine.